### PR TITLE
Goals Section

### PR DESCRIPTION
**Pull request by Dprof-code**
## Description:
- implemented the goals section

## Changes Made:
- implemented the HTML codes for the goals section
 - added the images for the goals section
- created the CSS file

## Related Issue:
- Fixes #[16]

## Screenshots (if applicable):

html-goal-section

@Algorithmia-SE-2023 
@ALX-SE-Algorithmia/code-reviewing-team

### CODE REVIEW

**Comment by Dprof-code** [comment]
@Hullaah @Bigizic @chisomchinaza @2023rahabu @Endygrit @ALX-SE-Algorithmia/code-reviewing-team @DestinedCodes I have done the HTML part of the goals section. I would start working on the stylings or should I let you guys do it?

**Comment by dohoudaniel** [comment]
Hi @Hullaah.
Thank you so much for your phenomenal work.
This is really awesome.

**Comment by dohoudaniel** [comment]
However, @Hullaah, for your PR to be merged, please make the following changes:
1. Delete the uploaded picture files. I sincerely appreciate your effort working on these images, but the pictures have been added to the repo already, in a PR. You can find them <a href="https://github.com/ALX-SE-Algorithmia/ALX-SE-Algorithmia.github.io/tree/main/images/home">here</a>.
2. Accordingly, you could edit your HTML file.

Once again, thank you very much.

File: images/goals-thumbnails/goal-1.jpg
Comment (dohoudaniel) [inline images/goals-thumbnails/goal-1.jpg:1]: Please delete this file and the repo (goals-thumbnails) accordingly.

File: images/goals-thumbnails/goal-2.jpg
Comment (dohoudaniel) [inline images/goals-thumbnails/goal-2.jpg:1]: Please delete this file and the repo (goals-thumbnails) accordingly.

File: images/goals-thumbnails/goal-3.jpg
Comment (dohoudaniel) [inline images/goals-thumbnails/goal-3.jpg:1]: Please delete this file and the repo (goals-thumbnails) accordingly.

File: images/goals-thumbnails/goal-4.jpg
Comment (dohoudaniel) [inline images/goals-thumbnails/goal-4.jpg:1]: Please delete this file and the repo (goals-thumbnails) accordingly.

File: images/goals-thumbnails/goal-5.jpg
Comment (dohoudaniel) [inline images/goals-thumbnails/goal-5.jpg:1]: Please delete this file and the repo (goals-thumbnails) accordingly.

File: images/goals-thumbnails/goal-6.jpg
Comment (dohoudaniel) [inline images/goals-thumbnails/goal-6.jpg:1]: Please delete this file and the repo (goals-thumbnails) accordingly.

File: index.html
Comment (dohoudaniel) [inline index.html:1]: Make edits to this file using the already uploaded pictures.

File: css/styles.css
Comment (dohoudaniel) [inline css/styles.css:1]: You can leave this file.

**Review by dohoudaniel — APPROVED** [review]

File: images/goals-thumbnails/goal-1.jpg
Comment (Dprof-code) [reply images/goals-thumbnails/goal-1.jpg:1]: I don't understand. I should delete the images?

File: index.html
Comment (Dprof-code) [reply index.html:1]: ohhh I didnt realise the images were uploaded already. I would do a git pull now and implement the necessary changes.

**Comment by Dprof-code** [comment]
@ALX-SE-Algorithmia/code-reviewing-team @dohoudaniel I have resolved the reviews, I would work on the stylings.

File: index.html
Comment (dohoudaniel) [reply index.html:1]: Thank you.

**Review by dohoudaniel — APPROVED** [review]

**Comment by dohoudaniel** [comment]
Hi @Dprof-code.
Thanks again for your PR.
However, please update your local branch to aid this PR.
Your code has been added to the final HTML file.
Thank you.

**Pull request closed without merging**